packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -70,6 +70,12 @@ export interface TimeIntervalNotificationTrigger {
   seconds: number;
 }
 
+export interface DailyNotificationTrigger {
+  type: 'daily';
+  hour: number;
+  minute: number;
```

Comment:
I don't think we have to add `repeats` here as there shouldn't be any ambiguity in the `NotificationTrigger` handling as the Calendar one is for iOS only and the Daily one is for Android only.

Reply:
True, I was torn between adding it and not (in case we ever added support for non-repeating daily trigger), but:
- going from `repeats: true` to `repeats: boolean` would be a breaking change even so
- I'm not sure if adding `repeats: boolean` now would make sense

I'll leave it up to you whether to add it or not. 🙂 (Let's remove it!)